include/tins/ip.h

```diff
@@ -776,12 +782,12 @@ class TINS_API IP : public PDU {
     void stream_options(Memory::OutputMemoryStream &stream, const uint32_t pad_size) const;
     void add_route_option(option_identifier id, const generic_route_option_type& data);
     generic_route_option_type search_route_option(option_identifier id) const;
-    void checksum(uint16_t new_check);
     options_type::const_iterator search_option_iterator(option_identifier id) const;
     options_type::iterator search_option_iterator(option_identifier id);
 
     options_type options_;
     ip_header header_;
+    bool auto_set_checksum;
```

Comment:
Add a trailing underscore to this variable. I think that the convention for this library is to use a trailing underscore to identify private member data.

Reply:
Whoops, the variable I initially introduced in my previous commits had underscore suffix, not sure why I didn't do that in my last commit.  Fixed